sections/results.tex

```diff
@@ -1,35 +1,23 @@
-These measures ignore learning, community building as well as networking outcomes, which are often equally important objectives.
-One possible approach here is to ask participants to self-report their success via a post-workshop survey.
-It is in principle possible to measure gains in knowledge, networks and productivity within the pool of acceptable candidates for both those that attended the hack week and those that did not.
+One key metric is the number of publications that result from hack week projects, but this is a fairly narrow definition of success, in line with standard academic performance indicators.
+Assuming that participants work largely in the open during a hack week, and that most projects have a strong programming component another indicator of success is the activity of participants in terms of code written and committed to a public code repository.
+Still, these measures ignore learning, community-building as well as networking outcomes.
+These outcomes can be assessed through post-workshop surveys.
+%It is in principle possible to measure gains in knowledge, networks and productivity within the pool of acceptable candidates for both those that attended the hack week and those that did not.
 %This would then provide a somewhat objective measure of the impact the hack week has had.
 %In practice, this has not been done for any of the hack weeks conducted thus far.
-For practical purposes, we recommend an approach that combines all of these metrics: publications, new code and projects generated, self-reported learning outcomes, and survey results.
+Here, we have taken an approach that combines these metrics: we start with survey results, and anecdotally report about publications, new code and projects generated (see \ref{box:outcomes})
 %Open-ended questions allow participants to provide feedback about outcomes, problems and goals that organizers had not anticipated.
 
-If hack week organizers plan to conduct research that involves hack week participants (for example, using assessments and evaluations in a paper such as this one) it is important to obtain approval from the Institutional Review Board (IRB), or equivalent body that approves research with human participants at the institution hosting the hack week.
-Though this research would usually fall under the category of ''minimal risk``, it is still important to establish procedures to manage these data, and to obtain informed consent.
-In particular, in some cases hack week organizers may be interested in studying not only the participants in the hack week, but also the applicants who did not end up participating.
+%If hack week organizers plan to conduct research that involves hack week participants (for example, using assessments and evaluations in a paper such as this one) it is important to obtain approval from the Institutional Review Board (IRB), or equivalent body that approves research with human participants at the institution hosting the hack week.
```

Comment:
Mentioning that we have IRB approval and informed consent is good (and I just added that in https://github.com/uwescience/HackWeek-Writeup/pull/60/commits/d32b4bee996b8ffd1460d5529e2acdc29cf8e677). I think that an more comprehensive explanation of what an IRB is and that organizers need IRB approval to conduct research on their participants is out of scope I believe. Note that IRB approval is *not required* for data collection that is intended for programmatic improvements (for example). Only for data collection for research purposes (for example, if you want to write a paper about your hack week). So I believe this will be of relevance to only a rather small group of people.